homeassistant/components/homewizard/strings.json

```diff
@@ -18,6 +18,10 @@
       "reauth_confirm": {
         "description": "The local API is disabled. Go to the HomeWizard Energy app and enable the API in the device settings."
       },
+      "authorize": {
+        "title": "Authorize",
+        "description": "Press the button on the HomeWizard Energy device, then click the button below."
```

Comment:
How many buttons does such device have?

Reply:
For the devices that support this flow exactly one. We assume users understand what button 'the button' is, as this button needs to be pressed during initial Wi-Fi setup.

In the future there are some exceptions, I think we can tackle those when we add support for these cases.

Reply:
To be sure, I've added some extra documentation: https://github.com/home-assistant/home-assistant.io/pull/37035